src/Symfony/Component/Routing/Tests/Fixtures/controller/routing.xml

```diff
@@ -6,6 +6,8 @@
 
     <route id="app_homepage" path="/" controller="AppBundle:Homepage:show" />
 
+    <route id="app_homepage_no_slash" path="" controller="AppBundle:Homepage:show" />
```

Comment:
Isn't this notation deprecated as of 4.1? Would make sense to add the non-deprecated notation while at it

Reply:
There are more in the fixtures, but we don't really care as that's not parsed anywhere in Routing, where it's just a string.

Reply:
Well it's this is just a routing fixture and the controller is not used as a controller. So you can write anything in there.

Reply:
we can even remove it, done :)